src/components/ItaliaTheme/View/CartellaModulisticaView/CartellaModulisticaView.jsx

```diff
@@ -68,21 +68,30 @@ const CartellaModulisticaView = ({ content }) => {
     );
   };
 
+  const filterItemsFN = (item) => {
+    return (
+      item?.title.toLowerCase().indexOf((searchableText ?? '').toLowerCase()) >=
```

Comment:
```suggestion
      item.title ? item.title.toLowerCase().indexOf((searchableText ?? '').toLowerCase()) >= 0 : true;
```

altrimenti se veramente item.title non è definito si romperebbe, perchè si cerca di applicare `toLowerCase` ad un undefined